src/client/utils/tracking/adtraction.ts

```diff
@@ -8,15 +8,17 @@ import {
   isDanishAccidentBundle,
   isStudentOffer,
   isDanishTravelBundle,
+  getMainQuote,
 } from 'pages/OfferNew/utils'
 import { NoComboTypes, DkBundleTypes } from './tracking'
 
+// Exclude single contract types since they are only sold as part of bundles
```

Comment:
Miiiinor change that might make it even clearer? 🙈 
```suggestion
// Exclude single contract types that are only sold as part of bundles
```